pkg/scheduler/framework/plugins/nodeunschedulable/node_unschedulable.go

```diff
@@ -84,10 +86,10 @@ func (pl *NodeUnschedulable) isSchedulableAfterPodTolerationChange(logger klog.L
 		// - Taint can be added, but can't be modified nor removed.
 		// - If the Pod already has the toleration, it shouldn't have rejected by this plugin in the first place.
 		//   Meaning, here this Pod has been rejected by this plugin, and hence it shouldn't have the toleration yet.
-		if v1helper.TolerationsTolerateTaint(modifiedPod.Spec.Tolerations, &v1.Taint{
+		if v1helper.TolerationsTolerateTaint(logger, modifiedPod.Spec.Tolerations, &v1.Taint{
 			Key:    v1.TaintNodeUnschedulable,
 			Effect: v1.TaintEffectNoSchedule,
-		}) {
+		}, utilfeature.DefaultFeatureGate.Enabled(features.TaintTolerationComparisonOperators)) {
```

Comment:
You shouldn't directly access to `utilfeature.DefaultFeatureGate.Enabled`, and instead you should access like pl.enableTaintTolerationComparisonOperators that other plugins do.